operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -205,7 +204,7 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
         MultivaluedMap<String, String> queryParametersWithoutDates = new MultivaluedHashMap<String,String>(queryParameters);
         boolean startOrEndProvided = queryParametersWithoutDates.remove(DATE_QUERY_PARAMETER) != null;
 
-        List<String> defaultResourceTypes = new ArrayList<String>(0);
+        List<String> defaultResourceTypes = new ArrayList<String>();
```

Comment:
there's no need to assign any value to defaultResourceTypes here. The value gets assigned in the next block, or an exception is thrown

Reply:
```suggestion
        List<String> defaultResourceTypes;
```